web/src/components/NewsListItem.tsx

```diff
@@ -46,14 +46,24 @@ const StyledContainer = styled.div`
 type NewsListItemProps = {
   title: string
   content: string
-  timestamp: Moment
+  timestamp: DateTime
   formatter: DateFormatter
+  format?: string
```

Comment:
This is never passed, is it?

Reply:
@steffenkleinle no, it is passed via **lastUpdate**

Reply:
Can we then just remove it?